newsletters/2024-09.md

```diff
@@ -0,0 +1,62 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2024-03.md)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter September 2024
+
+TL;DR: Watch the BOSC 2024 recordings, save the date for BOSC 2025, share or apply for the upcoming OBF event fellowship application and join the Pydna hackathon!
+
+In longer form: 
+
+## OBF News
+
+### BOSC news
+
+#### BOSC 2024: Watch the videos!
+
+This year we celebrated the 25th annual _Bioinformatics Open Source Conference_ (BOSC 2024), 
+which took place in July as part of ISMB 2024 alongside the post-conference _CollaborationFest_, 
+a collaborative work event (including but not limited to hacking).
+
+Our amazing conference organising committee has [written up a report for F1000 about the conference](https://docs.google.com/document/d/1P8Vzr_Rja7-yS686cL9TX7amoUfL5jt_qyH1wu3JGtw/edit),
```

Comment:
I don't want this to go out with the raw gdoc; we will hopefully have an F1000 URL before this newsletter goes out.

Reply:
Given that the one community item we have is somewhat time critical, I'd like this to go out by the end of the week if possible. If F1000 takes longer we could just remove that sentence for now and just link to the talks? :slightly_smiling_face:

Reply:
Oh, sorry, I thought we had until early Oct. Ok, then this could say
has written up a report for F1000 about the conference, which will be published soon in the [BOSC Collection](https://f1000research.com/collections/bosc)

Reply:
that sounds good!